index.js

```diff
@@ -1,9 +1,9 @@
-const { chromium } = require('playwright')
-const dayjs = require('dayjs')
-const customParseFormat = require('dayjs/plugin/customParseFormat')
-const config = require('./config.json')
+const { chromium } = require("playwright");
```

Comment:
```suggestion
const { chromium } = require('playwright');
```
Can we keep single quote for strings? I'll add linter soon to control project code format and help new developer onboarding

Reply:
Yep, no worries ✅